### DISCUSSION TITLE
About "AC/DC" coupling mode

### DISCUSSION THREAD

**Discussion by orientzane** [fhdm-dev/scoppy#129] [Q&A]
Hello
Does your designed oscilloscope APP have the "AC/DC" coupling mode? Will the screen simultaneously display the AC waveform and DC voltage straight line? If it supports, which analog front-end can support the "AC/DC" coupling mode?
Thanks

**Comment by fhdm-dev** [comment]
Hi. The app doesn't have any direct support for AC/DC coupling but you can easily implement it in hardware by adding a 100nF capacitor in series with the input for AC coupling and no capacitor for DC.
You can see an example on my [FSCOPE-500K](https://oshwlab.com/fruitloop57/fscope-250k5-v2_copy_copy_copy_copy) board. The instructions are [here](https://oscilloscope.fhdm.xyz/wiki/fscope-dso-500k-v2-usage) (see the AC/DC coupling section).
